experiments/functional/scale_jiva_replica/test.yml

```diff
@@ -14,6 +14,26 @@
        - include_tasks: /utils/fcm/update_litmus_result_resource.yml
          vars:
            status: 'SOT'
+       
+       - block:
+
+           - name: Getting the application pod name
+             shell: kubectl get pod -n {{ namespace }} -l {{ label }} -o jsonpath='{.items[0].metadata.name}'
+             register: app_pod_name
+
+           - name: Getting maya-api server pod name
+             shell: kubectl get pods -n {{ operator_namespace }} -l name=maya-apiserver --no-headers -o jsonpath='{.items[0].metadata.name}'
+             register: maya_api_pod
+
+           - name: Getting the application mount point
+             shell: kubectl get pod {{ app_pod_name.stdout }} -n {{ namespace }} -o jsonpath='{.spec.containers[0].volumeMounts[0].mountPath}'
+             register: app_mount_path
+
+             #Writing approx 1.6G of data at mount point
+           - name: Writing data on application mount point
+             shell: kubectl exec -it {{ app_pod_name.stdout }} -n {{ namespace }} -- sh -c "cd {{ app_mount_path.stdout }} && dd if=/dev/urandom of=test.txt bs=4k count=400024"
```

Comment:
run this task and induce controller failure parallely.

Reply:
@utkarshmani1997 Now we are launching dd-client for continuos I/Ss.

Reply:
@shashank855 let's remove this parallel io as it is not required as of now. I got confused this PR with the optimization scenario.